src/stravalib/tests/unit/test_validate_activity_type.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+
+@pytest.fixture
+def params():
+    """A fixture that creates a dictionary representing
+    parameters associated with a Strava activity."""
+
+    params = {
+        "name": "New Activity",
+        "start_date_local": "2024-03-04T18:56:47Z",
+        "elapsed_time": 9000,
+        "description": "An activity description here.",
+        "distance": 5700,
+    }
+    return params
```

Comment:
👍, this is much better than the `default_call_kwargs` etc that I used in `test_create_activity`. Feel free to replace these with fixtures as well!